crates/sncast/src/starknet_commands/declare.rs

```diff
@@ -68,7 +68,7 @@ pub async fn declare(
         .context("Failed to get scarb metadata")?;
 
     let package = get_package_metadata(&metadata, &manifest_path)
-        .with_context(|| anyhow!("Failed to find package for contract = {contract_name}"))?;
+        .with_context(|| anyhow!("Failed to find package for manifest path = {manifest_path}"))?;
```

Comment:
`manifest_path` may not lead to the `contract_name` package - for example in the case of deployment scripts

Reply:
Please get familiar with this PR stack https://github.com/foundry-rs/starknet-foundry/pull/1565
I think this PR can be closed.